frontend/src/components/FilterForm/FilterForm.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import styles from './FilterForm.module.css';
```

Comment:
Thanks Petr for the review..I will do responsiveness and apply form  in the current week as a new task in meister.. For apply form I did some structuring with dummy data as I don't want to waste time till routes are ready to shared.It would be nice if I can share my code with teams so that they can also work on the filter component , apply form and styling.
The files to be reviewed are JobList , JobCard, JobDetail.. Fixing the styling for responsiveness will be doing this week.

Reply:
Hey @RemyaManoharan! ok so if I understand it correctly, you are supposed to do only the "skeleton" of the components, right? If so, please create a ticket for each component that needs the stylings fix and responsivity before we can proceed with the review so I know that the work will be done :) Thank you 🙏

Also some small things:
1. I noticed that you shared some code with @sayali05 's PR, can you remove everything that is not related to the components? I reviewed the Backend work as a part of this [PR](https://github.com/HackYourFuture-CPH/class-25-ironside/pull/17) already

2. Please check and resolve the lining warnings 🙏 You can see them if you click on "files changed" tab here on github. It should be just simple removing unused code. 
![Screenshot 2023-10-02 at 9 20 37](https://github.com/HackYourFuture-CPH/class-25-ironside/assets/48878547/a2132970-d897-4a01-b4c2-79d040f68eff)

3. I noticed that fonts are off, will we add them in a separate PR?

Thank you and GJ! ❤️

Reply:
Thanks Petr.. I will be resolve lining warning and pushing again.. I need Sayali's backend routes to fetch jobs using jobstore.tsx and to render the jobs using JobCards and JobDetail.. So If I delete the Sayalis code I will be getting errors.

Reply:
Oki @RemyaManoharan , understand.. Then please resolve all comments and linting errors, and I will make sure that there are no conflicts after that we can merge is it ok ?:) also please share the newly created tickets for each component that needs the new styling here so I can refer to it in the future 🙏. Could you create a ticket for adding fonts for me, please? Thank you! ❤️

Reply:
Ok.. I will be doing all the mentioned and will be pushing again.. Thanks :)